Tasks/NotationV0/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "notation",
+  "version": "0.1.0",
+  "description": "Azure Pipepine Task for setting up Notation CLI, sign and verify with Notation",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "tsc -b"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@types/node": "^20.3.1",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "4.6.1",
+    "azure-pipelines-tool-lib": "2.0.7"
+  },
+  "devDependencies": {
+    "typescript": "4.0.2"
```

Comment:
afaik for node20 tasks you need to set ["typescript": "5.1.6"](https://github.com/microsoft/azure-pipelines-tasks/blob/master/_generated/AzureAppServiceSettingsV1_Node20/package.json#L33)

@qianz2, could you please check as well, as more proficient in node 20 migration?

Reply:
Typescript or @type/node versions in package.json files would be automatically updated thru BuildConfigGen for node20 tasks, so we don't need to worry about it in base tasks for now.

Reply:
Will BuildConfigGen automatically generate Node20 version for the task? @qianz2

Reply:
Sorry for the late reply, yes buildConfigGen will automatically generate node20 tasks.